frontend/TestRun/Jenkins/SCTParameterWizard.svelte

```diff
@@ -0,0 +1,414 @@
+                    internalName: "scylla_version",
+                    condition: (params, defs) => (defs.scyllaVersion.currentSource == "scylla_version"),
+                    onChange: function (e, params) {
+                        params.scylla_repo = "";
+                        params.scylla_ami_id = "";
+                        params.gce_image_db = "";
+                        params.azure_image_db = "";
+                    }
+                },
+                scyllaRepo: {
+                    name: "Scylla Repository",
+                    description: "HTTP(s) link to the repo file",
+                    type: StringParam,
+                    internalName: "scylla_repo",
+                    condition: (params, defs) => (defs.scyllaVersion.currentSource == "scylla_repo"),
+                    onChange: function (e, params) {
+                        params.scylla_version = "";
+                        params.scylla_ami_id = "";
+                        params.gce_image_db = "";
+                        params.azure_image_db = "";
+                        this.validated = this.validate(params);
+                    }
+                },
+                scyllaImageAWS: {
+                    name: "Scylla AMI",
+                    description: "AWS Machine Image id",
+                    type: StringParam,
+                    internalName: "scylla_ami_id",
+                    condition: (params, defs) => (defs.scyllaVersion.currentSource == "scylla_image" && inBackendGroup(params.backend, "aws")),
+                    onChange: function (e, params) {
+                        params.scylla_version = "";
+                        params.scylla_repo = "";
+                    }
+                },
+                scyllaImageGCE: {
+                    name: "Scylla GCE Image",
+                    description: "GCE ImageId",
+                    type: StringParam,
+                    internalName: "gce_image_db",
+                    condition: (params, defs) => (defs.scyllaVersion.currentSource == "scylla_image" && inBackendGroup(params.backend, "gce")),
+                    onChange: function (e, params) {
+                        params.scylla_version = "";
+                        params.scylla_repo = "";
+                    }
+                },
+                scyllaImageAzure: {
+                    name: "Scylla Azure ImageId",
+                    description: "Azure ImageId URI",
+                    type: StringParam,
+                    internalName: "azure_image_db",
+                    condition: (params, defs) => (defs.scyllaVersion.currentSource == "scylla_image" && inBackendGroup(params.backend, "azure")),
+                    onChange: function (e, params) {
+                        params.scylla_version = "";
+                        params.scylla_repo = "";
+                    }
+                },
+                updateDbPackages: {
+                    name: "Package Update URI",
```

Comment:
this a bit opaque, and missing the reference to DB 
again this need the explanation, otherwise no one would understand what it is